src/routes/edit/share-options/exports/frameworks/angular/fragment-v10.ts

```diff
@@ -75,6 +76,22 @@ const getComponentCode = (fragment: any, fragments: any[]) => {
 	return componentCode;
 };
 
+const checkForPlaceholder = (json: any) => {
+	for (const component of Object.values(allComponents)) {
+		if (json.type === component.componentInfo.type && component.componentInfo.codeExport.angular?.usePlaceholder) {
+			return true;
+		}
+	}
```

Comment:
Can this be rewritten with `some` or something similar?

Reply:
simplified